asab/library/providers/git.py

```diff
@@ -176,6 +176,7 @@ def init_task():
 		except Exception as err:
 			L.exception(err)
 
+		assert hasattr(self.GitRepository, "remotes"), "Git repository not initialized."
```

Comment:
it can also happen that self.GitRepository is None

Reply:
But that is handled by `hasatrr()`, isn't it?
```python
>>> hasattr(None, "kocka")
False
```